common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt

```diff
@@ -62,7 +70,19 @@ interface INativeAuthCommandResult {
         SignUpSubmitCodeCommandResult, SignUpResendCodeCommandResult,
         SignUpSubmitPasswordCommandResult,
         ResetPasswordStartCommandResult, ResetPasswordSubmitCodeCommandResult,
-        ResetPasswordResendCodeCommandResult, ResetPasswordSubmitNewPasswordCommandResult
+        ResetPasswordResendCodeCommandResult, ResetPasswordSubmitNewPasswordCommandResult {
+        override fun toSafeString(mayContainPii: Boolean): String {
+            return if (mayContainPii) {
```

Comment:
Should it not be the other way round? If the object may contain PII then we just print the `correlationId`. If the object definitley does not contain PII then we print other details as well.

Reply:
`mayContainPii` doesn't state whether the object contains PII. Rather it states whether the string that `toSafeString()` returns may contain PII. In the case of `true`, we can use PII fields like `error`. In the case of `false`, only `correlationId` is safe to return.

I will add comments explaining the role of these methods and parameteters.